documentation-samples/multinic/azuredeploy.json

```diff
@@ -0,0 +1,287 @@
+      "defaultValue": "NSG-RemoteAccess",
+      "metadata": {
+        "description": "Name for the NSG used to allow remote access."
+      }
+    },
+    "stdStorageName": {
+      "type": "string",
+      "defaultValue": "wtestvnetstoragestd",
+      "metadata": {
+        "description": "Name for simple storage account."
+      }
+    },
+    "prmStorageName": {
+      "type": "string",
+      "defaultValue": "wtestvnetstorageprm",
+      "metadata": {
+        "description": "Name for simple storage account."
+      }
+    },
+    "prmStorageType": {
+      "type": "string",
+      "defaultValue": "Premium_LRS",
+      "allowedValues": [
+        "Premium_LRS",
+        "Premium_GRS",
+        "Premium_ZRS"
+      ]
+    },
+    "osType": {
+      "type": "int",
+      "defaultValue": 0,
+      "allowedValues": [
+        0,
+        1
+      ],
+      "metadata": {
+        "description": "Type of OS to use for VMs: 0 - Windows; 1 - Ubuntu."
+      }
+    },
+    "dbCount": {
+      "type": "int",
+      "defaultValue": 2,
+      "metadata": {
+        "description": "Number of database VMs to be deployed to the backend end subnet."
+      }
+    },
+    "parentRG": {
+      "type": "string",
+      "defaultValue": "IaaSStory",
+      "metadata": {
+        "description": "Name of resource group containing vnet and front end resources."
+      }
+    }
+  },
+  "variables": {
+    "dbVMSettings": [
+      {
+        "vmSize": "Standard_DS3",
+        "adminUsername": "adminuser",
+        "adminPassword": "adminP@ssw0rd",
+        "publisher": "MicrosoftSQLServer",
+        "offer": "SQL2014SP1-WS2012R2",
+        "sku": "Standard",
+        "version": "latest",
+        "vmName": "DB",
+        "osdisk": "osdiskdb",
+        "datadisk": "datadiskdb",
+        "nicName": "NICDB",
+        "ipAddress": "192.168.2.",
+        "extensionDeployment": "",
+        "avsetName": "ASDB",
+        "remotePort": 3389,
+        "dbPort": 1433
+      },
+      {
+        "vmSize": "Standard_DS3",
+        "adminUsername": "adminuser",
+        "adminPassword": "adminP@ssw0rd",
+        "publisher": "Canonical",
+        "offer": "UbuntuServer",
+        "sku": "14.04.2-LTS",
+        "version": "latest",
+        "vmName": "DB",
+        "osdisk": "osdiskdb",
+        "datadisk": "datadiskdb",
+        "nicName": "NICDB",
+        "ipAddress": "192.168.2.",
+        "extensionDeployment": "",
+        "avsetName": "ASDB",
+        "remotePort": 22,
+        "dbPort": 1433
+      }
+    ],
+    "location": "[resourceGroup().location]",
+    "vmStorageAccountContainerName": "vhds",
+    "vnetID": "[resourceId(parameters('parentRG'),'Microsoft.Network/virtualNetworks',parameters('vnetName'))]",
+    "backEndSubnetRef": "[concat(variables('vnetID'),'/subnets/',parameters('backEndSubnetName'))]",
+    "dbVMSetting": "[variables('dbVMSettings')[parameters('osType')]]"
+  },
+  "resources": [
+    {
+      "apiVersion": "2015-05-01-preview",
```

Comment:
All awesome feedback. I will implement them today. One quick question, where can I find the latest schema? I use VS2015 to create my templates, and it currently points to an older schema.

From: Kay Singh [mailto:notifications@github.com]
Sent: Friday, October 30, 2015 11:32 AM
To: Azure/azure-quickstart-templates <azure-quickstart-templates@noreply.github.com>
Cc: Telmo Sampaio <telmos@microsoft.com>
Subject: Re: [azure-quickstart-templates] Added multiNIC documentation sample (#768)


In documentation-samples/multinic/azuredeploy.json<https://na01.safelinks.protection.outlook.com/?url=https%3a%2f%2fgithub.com%2fAzure%2fazure-quickstart-templates%2fpull%2f768%23discussion_r43536933&data=01%7c01%7ctelmos%40microsoft.com%7cc01a8655e38542ae904008d2e1586141%7c72f988bf86f141af91ab2d7cd011db47%7c1&sdata=%2bibOu%2bWLevgWqjtuEjulh7r4ut6c1BljOge3uVqkPTo%3d>:

> +        "ipAddress": "192.168.2.",

> +        "extensionDeployment": "",

> +        "avsetName": "ASDB",

> +        "remotePort": 22,

> +        "dbPort": 1433

> +      }

> +    ],

> +    "location": "[resourceGroup().location]",

> +    "vmStorageAccountContainerName": "vhds",

> +    "vnetID": "[resourceId(parameters('parentRG'),'Microsoft.Network/virtualNetworks',parameters('vnetName'))]",

> +    "backEndSubnetRef": "[concat(variables('vnetID'),'/subnets/',parameters('backEndSubnetName'))]",

> +    "dbVMSetting": "[variables('dbVMSettings')[parameters('osType')]]"

> +  },

> +  "resources": [

> +    {

> +      "apiVersion": "2015-05-01-preview",

Storage accounts work with 2015-06-15 API now

—
Reply to this email directly or view it on GitHub<https://na01.safelinks.protection.outlook.com/?url=https%3a%2f%2fgithub.com%2fAzure%2fazure-quickstart-templates%2fpull%2f768%2ffiles%23r43536933&data=01%7c01%7ctelmos%40microsoft.com%7cc01a8655e38542ae904008d2e1586141%7c72f988bf86f141af91ab2d7cd011db47%7c1&sdata=R6JjH9k2tihR8wa4bicUinbdLXM0ej%2fPk4nqVlzXPz0%3d>.